rocketpy/simulation/flight_data_importer.py

```diff
@@ -0,0 +1,399 @@
+"""Starts with .csv or .txt file containing the rocket's collected flight data
+and build a rocketpy.Flight object from it.
+"""
+import warnings
+from os import listdir
+from os.path import isfile, join
+
+import numpy as np
+
+from rocketpy.mathutils import Function
+from rocketpy.units import UNITS_CONVERSION_DICT
+
+FLIGHT_LABEL_MAP = {
+    # "name of Flight Attribute": "Label to be displayed"
+    "acceleration": "Acceleration (m/s^2)",
+    "alpha1": "Angular acceleration x (rad/s^2)",
+    "alpha2": "Angular acceleration y (rad/s^2)",
+    "alpha3": "Angular acceleration z (rad/s^2)",
+    "altitude": "Altitude AGL (m)",
+    "ax": "Acceleration x (m/s^2)",
+    "ay": "Acceleration y (m/s^2)",
+    "az": "Acceleration z (m/s^2)",
+    "bearing": "Bearing (deg)",
+    "density": "Density (kg/m^3)",
+    "drift": "Drift (deg)",
+    "e0": "Quaternion e0",
+    "e1": "Quaternion e1",
+    "e2": "Quaternion e2",
+    "e3": "Quaternion e3",
+    "latitude": "Latitude (deg)",
+    "longitude": "Longitude (deg)",
+    "mach_number": "Mach number",
+    "name": "Time (s)",
+    "speed": "Speed (m/s)",
+    "stability_margin": "Stability margin",
+    "static_margin": "Static margin",
+    "vz": "Velocity z (m/s)",
+    "vx": "Velocity x (m/s)",
+    "vy": "Velocity y (m/s)",
+    "w1": "Angular velocity x (rad/s)",
+    "w2": "Angular velocity y (rad/s)",
+    "w3": "Angular velocity z (rad/s)",
+    "x": "Position x (m)",
+    "y": "Position y (m)",
+    "z": "Position z (m)",
+}
+
+
+class FlightDataImporter:
+    """A class to create a rocketpy.Flight object from a .csv file. The data
+    frame must contain a time column in seconds.
+
+    There are different ways of calculating a property from the data frame.
+        1. The property is already in data frame and you want to use it as is
+        2. The property is not in the data frame but you can calculate it from
+            other properties in the data frame.
+
+    Initially, this class is designed to work with the option '1.'
+    """
+
```

Comment:
I would say "outdated" instead of "wrong".

Solved here: [4a473d6](https://github.com/RocketPy-Team/RocketPy/pull/474/commits/4a473d6a7a5bb48057917b308e37b39244e8e889)